web_generator/lib/src/ast/builtin.dart

```diff
@@ -51,7 +46,7 @@ class BuiltinType extends Type {
           .where((p) => typeParams.length != 1 || p != $voidType)
           .map((p) => p.emit(options)))
       ..url = fromDartJSInterop ? 'dart:js_interop' : null
-      ..isNullable = _isNullable ?? options?.nullable);
+      ..isNullable = isNullable || (options?.nullable ?? false));
```

Comment:
In theory, `isNullable` should have been set so we should not see any late initialization errors with the propose change above.